schema.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(225),
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories(
+    id INT  PRIMARY KEY,
+    admited_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(225)
+);
+
+CREATE TABLE envoices (
+    id INT PRIMARY KEY,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE traitments(
+    id INT PRIMARY KEY,
+    type VARCHAR(225),
+    name VARCHAR(225)
+);
```

Comment:
- [OPTIONAL] The correct name is `treatments` and not `traitments`. It would be nice to use the correct spelling to show case your professionalism. 👍 😎